addons/website_autocomplete/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# -*- encoding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'ECommerce Autocompletion',
+    'category': 'Website/Website',
+    'summary': 'Assist your users with automatic completion & suggestions when filling forms',
+    'version': '1.0',
+    'description': "Assist your users with automatic completion & suggestions when filling forms",
+    'depends': [
+        'website_sale',
```

Comment:
Since the module depends on `website_sale` it's more logical to have it named `website_sale_autocomplete`